robot_id: 'robot_1'

cam_params:
  focal_length: 1.3 # f = 1/d, d being the distance of focus in [meters]. So a focal length of 2.0 corresponds a net image at 0.5m of distance 
  dist_cam_robot_center: 0.102 #meters
  aruco_square_size: 0.038 #meters

aruco_params:
  334:
    t_x: 0.123
    t_y: 0.456
  34:
    t_x: 1.234
    t_y: 2.345
  416:
    t_x: 1.0
    t_y: 1.0
  618:
    t_x: 4.395
    t_y: 7.358
  11:
    t_x: 4.3125
    t_y: 6.368
  645:
    t_x: 4.35
    t_y: 5.463
  614:
    t_x: 4.37
    t_y: 4.51